grails-app/controllers/org/pih/warehouse/picklist/PicklistController.groovy

```diff
@@ -20,6 +20,7 @@ class PicklistController {
 
     def picklistService
     def pdfRenderingService
+    def localizationService
```

Comment:
is it still needed?

Reply:
That's what I'm talking about! This is the kind of nitpicky review comments I'm looking for.